src/components/Tooltip/BaseTooltip.tsx

```diff
@@ -183,15 +194,15 @@ function Tooltip({children, numberOfLines, maxWidth, text, renderTooltipContent,
                     yOffset={yOffset}
                     targetWidth={wrapperWidth}
                     targetHeight={wrapperHeight}
-                    shiftHorizontal={Str.result(shiftHorizontal)}
-                    shiftVertical={Str.result(shiftVertical)}
+                    shiftHorizontal={shiftHorizontal}
+                    shiftVertical={shiftVertical}
```

Comment:
We have to add a new `result` function which returns a number instead of a string.